src/App.test.js

```diff
@@ -1,8 +1,32 @@
 import { render, screen } from '@testing-library/react';
 import App from './App';
+import userEvent from '@testing-library/user-event';
 
-test('renders learn react link', () => {
+test('test for adding, editing and deleting in list', () => {
   render(<App />);
-  const linkElement = screen.getByText(/learn react/i);
-  expect(linkElement).toBeInTheDocument();
+
+  const itemInput = screen.getByRole('textbox');
+  expect(itemInput).toBeInTheDocument();
+
+  userEvent.type(itemInput, 'Oatmilk');
+  expect(itemInput.value).toBe('Oatmilk');
+
+  const addButton = screen.getByRole('button', { name: 'Add' });
+  expect(addButton).toBeInTheDocument();
+  userEvent.click(addButton);
+
+  const deleteButton = screen.getByLabelText('Delete Jalapenos');
+  userEvent.click(deleteButton);
+  expect(screen.queryByText(' Jalapenos ')).not.toBeInTheDocument();
```

Comment:
This is going to look explicitly for ' Jalapenos '. Not 'Jalapenos' or 'jalapenos'. We should use a regex here. 
queryByText(/delete jalapenose/i).toBeNull();
OR -- you could do queryByText('Jalapenos', { exact: false })  -- this should look for a substring match.
See (https://testing-library.com/docs/queries/about/#textmatch)
Also, queryBy pairs with .toBeNull  ( see https://testing-library.com/docs/queries/about/#priority)